### PR TITLE
HXL support.

### PR DESCRIPTION
**Pull request by urbanslug**
New post on HXL support. Have a question on the real life usages.

### CODE REVIEW

File: _posts/2016-06-02-full-HXL-support.md [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+This makes it much easier to know what exactly you are selecting.
+
+![HXL create filtered dataset](/assets/images/2016-06-02/create-filtered-dataset.png)
+
+
+
+## Real life usage of HXL.
+
+#### DFID Somalia.
+HXL is currently being used in data collection in Somalia to...
```
Comment (urbanslug) [inline _posts/2016-06-02-full-HXL-support.md]: @pld Can I get some help with this paragraph please.

File: _posts/2016-06-02-full-HXL-support.md [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+Since Ona is used in a lot of humanitarian data collection efforts
+we now support a standard for humanitarian data exchange known as the
+Humanitarian eXchange Language, in short HXL.
+
+What HXL does is that it enables easy aliasing of fields labels by letting
+us add more general names for our fields in a record underneath the header record.
+
+
+## How to use HXL.
+When authouring an XLS form add an extra record under the header record.
```
Comment (pld) [inline _posts/2016-06-02-full-HXL-support.md]: "author", also I think "row" should replace "record" throughout

Comment (urbanslug) [inline _posts/2016-06-02-full-HXL-support.md]: Ok, nice. I'll remember to use American English.

Comment (pld) [inline _posts/2016-06-02-full-HXL-support.md]: we just call it English

**Comment by urbanslug** [comment]
@pld Love the grammar watch 👍

File: _posts/2016-06-02-full-HXL-support.md [hunk 3]
```diff
@@ -20,56 +20,48 @@ we now support a standard for humanitarian data exchange known as the
 Humanitarian eXchange Language, in short HXL.
 
 What HXL does is that it enables easy aliasing of fields labels by letting
-us add more general names for our fields in a record underneath the header record.
+us add more general names for our fields in a how underneath the column header row.
```
Comment (urbanslug) [inline _posts/2016-06-02-full-HXL-support.md]: You mean row not how.

Comment (pld) [inline _posts/2016-06-02-full-HXL-support.md]: Ah yes

File: _posts/2016-06-02-full-HXL-support.md (hunk 1 above)
Comment (pld) [inline _posts/2016-06-02-full-HXL-support.md]: I don't know anything about this, @smusula?

**Comment by rowo** [comment]
@urbanslug @pld I think you can remove the real life use cases and update the date, and publish this.

**Comment by urbanslug** [comment]
@rowo Why'd we have to get rid of real life HXL use cases?

**Comment by rowo** [comment]
Talked about it on Slack. Reason is to publish it without delay. We'll do a follow up with a story.

**Comment by pld** [comment]
Because there's no content there

**Comment by urbanslug** [comment]
Yeah we talked about it on Slack.

**Pull request merged**